docs/css/fui-site.css

```diff
@@ -3356,7 +3356,7 @@ This will render 4 boxes spanning 2 cols each indented 2 cols
     display: none; }
   .fd-alert__close {
     position: absolute;
-    right: 4px;
```

Comment:
I think you want to make this change here: https://github.com/SAP/fundamental/blob/master/scss/components/alert.scss#L50